src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -303,13 +303,6 @@ public static boolean canEntityUpdate(Entity entity)
         return event.canUpdate;
     }
 
-    public static String onPlaySoundAt(Entity entity, String name, float volume, float pitch)
```

Comment:
What do you propose could be done instead? The method does not have the possibility to return all the necessary parameters. If it is going to stay there, it won't have much use since it cannot be properly called from anywhere.